src/app/project/project-details-tab/project-details-tab.component.html

```diff
@@ -47,6 +47,15 @@
       </div>
     </div>
     <br>
+    <div class="row">
```

Comment:
I think this would look cleaner if this div was hidden when there are no featured docs for the projects. could we do an ngIf here to check that?

Reply:
Yeah, good catch. Changed.